lib/integrated-service-sdk/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check
```

Comment:
Is there any reason behind using `lib/integrated-service-sdk` location for this Go module as opposed to putting it into `pkg/sdk/integrated-service` similarly to other SDKs?